app/templates/search.hbs

```diff
@@ -10,12 +10,12 @@
 <div class="div-search-bar">
 {{search-bar
   placeholder="Search For Jobs"
-  image="https://png.icons8.com/search-filled/ios7/39/00B5AF"
+  image="https://s3-us-west-1.amazonaws.com/waldojobs-dev/app-assets/icons/waldoSearch.png"
```

Comment:
I tried to use the icon asset helper here but could not get it to work.

Reply:
I got it to work by doing `image=(icon-asset "waldoSearch.png")`.

Its weird... When you are working in the context of an ember component, you use helpers by wrapping them in parentheses. When you are working in a standard html element, you use helpers by wrapping them with {{ }}